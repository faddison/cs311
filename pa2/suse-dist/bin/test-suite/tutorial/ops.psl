defvar o = {x:"str"} in
defvar p = {x:1} in {
print(<(3,4));
print("\n");
print(>(4,5));
print("\n");
print(+(3,4,5));
print("\n");
print(-(9,8,7));
print("\n");
print(+("one ", "two ", "three"));
print("\n");
print(==(5,5));
print("\n");
print(==("foo",5));
print("\n");
print(<(3,3));
print("\n");
print(+("one ", "two ", o.x));
print("\n");
print(>(3,3));
print("\n");
print(>(-(5,4),3));
print("\n");
print(<(-(5,4),3));
print("\n");
print(<(-(5,4),-(6,4)));
print("\n");
print(>(p.x,3));
print("\n");
print(<(p.x,3));
print("");
}
