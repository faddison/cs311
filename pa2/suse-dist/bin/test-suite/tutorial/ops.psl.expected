true
false
12
-6
one two three
true
false
false
one two str
false
false
true
true
false
true
